### DISCUSSION TITLE
Add DBIO to core

### DISCUSSION THREAD

**Discussion by torvista** [zencart/zencart#6733] [Ideas]
It's mature, a clean plugin and bug-free, what's not to like?

**Comment by lat9** [comment]
It needs to be "encapsulated", first!

**Comment by torvista** [comment]
I thought it was, and in any case, does that matter if it's core code?

**Comment by lat9** [comment]
Not everyone has a use for it.  Some people use EP and that's fine for them.

**Comment by torvista** [comment]
Hmm, I think import-export of some data is something most users will need at some point, while others may use constantly, and so its not a fringe functionality.
Since the codebase of DBIO is written to allow expansion, it seems a good base to have in-built for those who want to build on that for an api-like automated import-export with the outside world.
EP has a long and chequered history but I think it should be considered as retired in the light of the better solution, if only in the interests of consolidating developer and support efforts,
